### PR TITLE
Enhanced the pop up's display #29

### PR DESCRIPTION
**Pull request by prithaupadhyay**
image

@aashutoshrathi Please have a look at it. Looking forward to getting your feedback.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import url("https://fonts.googleapis.com/css?family=Montserrat");
 
 body {
-  height: 300px;
+  height: 350px;
```
Comment (aashutoshrathi) [inline style.css]: Make it 330px.

File: popup.html [hunk 2]
```diff
@@ -16,6 +16,15 @@ <h1 id="title">Instagram Downloader</h1>
         <p><b>DP :</b> Downloads the profile picture of the user</p>
         <p><b>Download:</b> Downloads the picture</p>
```
Comment (aashutoshrathi) [inline popup.html]: Downloads the picture/video.

File: style.css [hunk 3]
```diff
@@ -18,6 +18,10 @@ body {
   font-size: 30px;
 }
 
+#badges{
+  margin-left: 35px;
```
Comment (aashutoshrathi) [inline style.css]: Better if we center it.

File: style.css (hunk 1 above)
Comment (prithaupadhyay) [reply style.css]: On it 👍

**Comment by prithaupadhyay** [comment]
@aashutoshrathi  Please do have a look at the changes.
image

**Comment by aashutoshrathi** [comment]
That looks better :heart: 
Few suggestions:
- Move the badges below heading wrapped under `<hr>`s

**Comment by prithaupadhyay** [comment]
@aashutoshrathi  
image
Please do have a look.

**Comment by aashutoshrathi** [comment]
Thanks @prithaupadhyay

**Pull request merged**